app/api/helpers/files.py

```diff
@@ -129,8 +130,17 @@ def create_save_image_sizes(image_file, image_sizes_type, unique_identifier=None
     try:
         image_sizes = ImageSizes.query.filter_by(type=image_sizes_type).one()
     except NoResultFound:
-        image_sizes = ImageSizes(image_sizes_type, 1300, 500, True, 100, 75, 30, True, 100, 500, 200, True, 100)
-
+        if (image_sizes_type == 'speaker-image'):
+            image_sizes = ImageSizes(image_sizes_type, icon_size_width_height=35, icon_size_quality=80,
+                                     small_size_width_height=50, small_size_quality=80,
+                                     thumbnail_size_width_height=500, thumbnail_quality=80)
+        else:
+            image_sizes = ImageSizes(image_sizes_type, full_width=1300,
+                                     full_height=500, full_aspect=True, full_quality=80,
+                                     icon_width=75, icon_height=30, icon_aspect=True,
+                                     icon_quality=80, thumbnail_width=500, thumbnail_height=200,
+                                     thumbnail_aspect=True, thumbnail_quality=80, logo_width=500,
+                                     logo_height=200)
```

Comment:
No need for condition, add all values unconditionally

Reply:
The values are different for speaker-type and event-type

Reply:
No, they are mutually exclusive